8/community/Dockerfile

```
@@ -21,6 +18,8 @@ EXPOSE 9000
 RUN groupadd -r sonarqube && useradd -r -g sonarqube sonarqube
 
 SHELL ["/bin/bash", "-c"]
+RUN sed -i -e "s/securerandom.source=file:\/dev\/random/securerandom.source=file:\/dev\/urandom/g" \
```

Have you considered using an alternative pattern separator of `s///`, for example `s???` or `s|||` ?
```suggestion
RUN sed -i -e "s?securerandom.source=file:/dev/random?securerandom.source=file:/dev/urandom?g" \
```

---

I didn't know this one even a feature. I hardly did more than make my own sed from various sources on the web and all use `|`